esp-hal-common/src/gpio.rs

```diff
@@ -1510,12 +1380,12 @@ macro_rules! gpio {
 
             pub struct Pins {
                 $(
-                    pub [< gpio $gpionum >] : GpioPin<Unknown, [< Bank $bank GpioRegisterAccess >], $crate::gpio::[< InteruptStatusRegisterAccessBank $bank >], [< $type PinType >], [<Gpio $gpionum Signals>], $gpionum>,
+                    pub [< gpio $gpionum >] : GpioPin<Unknown, $gpionum>,
                 )+
             }
 
             $(
-                pub type [<Gpio $gpionum >]<MODE> = GpioPin<MODE, [< Bank $bank GpioRegisterAccess >], $crate::gpio::[< InteruptStatusRegisterAccessBank $bank >], [< $type PinType >], [<Gpio $gpionum Signals>], $gpionum>;
+                pub type [<Gpio $gpionum >]<MODE> = GpioPin<MODE, $gpionum>;
```

Comment:
I suppose we can remove these type alias' now that that concrete type is simplified.

Reply:
Oh never mind, that would definitely be a mistake :D.